tests/models/recurrent_gemma/test_modeling_recurrent_gemma.py

```diff
@@ -422,7 +387,20 @@ class RecurrentGemmaIntegrationTest(unittest.TestCase):
 
     @require_read_token
     def test_2b_generate(self):
-        EXPECTED_TEXTS = ['Hello I am doing a project on the topic of "The impact of the internet on the society" and I am looking for some information on the topic. I am looking for some information on the impact of the internet on the society. I am looking for some information on the impact of the internet on the society. I am looking for some', 'Hi today is a very good day for you. You will be able to do all the work you want to do. You will be able to do all the work you want to do. You will be able to do all the work you want to do. You will be able to do all the work you want to do.']  # fmt: skip
+        EXPECTED_TEXTS = [
```

Comment:
If we DON'T pass `position_ids` from `RecurrentGemmaForCausalLM` to the model itself, as before this PR, the tests pass without changes.

This update reflects the bugfix. Note that the 1st sequence (unpadded) doesn't change :)